src/DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,24 @@ final class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        return new TreeBuilder('monsieurbiz_sylius_nocommerce');
+        $treeBuilder = new TreeBuilder('monsieurbiz_sylius_nocommerce');
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            $rootNode = $treeBuilder->getRootNode();
+        } else {
+            // BC layer for symfony/config 4.1 and older
+            $rootNode = $treeBuilder->root('monsieur_biz_sylius_search');
+        }
```

Comment:
```suggestion
        $rootNode = $treeBuilder->getRootNode();
```

Sylius requires `"symfony/config": "^4.4"` on tag 1.8.0.